source/features/reactions-avatars.tsx

```diff
@@ -37,11 +37,18 @@ function add(): void {
 		const flatParticipants = flatZip(participantByReaction, avatarLimit);
 
 		for (const participant of flatParticipants) {
-			participant.container.append(
-				<a href={`/${participant.username}`}>
-					<img src={`/${participant.username}.png?size=${window.devicePixelRatio * 20}`}/>
+			const avatar = (
+				<a>
+					<img src={`/${participant.username}.png?size=${window.devicePixelRatio * 20}`} />
 				</a>
 			);
+
+			// Without this, Firefox will follow the link instead of submitting the reaction button
+			if (!navigator.userAgent.includes('Firefox/')) {
+				(avatar as any).href = `/${participant.username}`
```

Comment:
Sadly `as HTMLAnchorElement` causes:

```ts
      TS2352: Conversion of type 'Element' to type 'HTMLAnchorElement' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'Element' is missing the following properties from type 'HTMLAnchorElement': charset, coords, download, hreflang, and 20 more.
```

And I don't remember if we came up with a better solution for this